### ISSUE TITLE
Error due to upgrading xss from 1.0.11 to 1.0.13 without changing the vis-timeline version from 6.5.1 ?

### ISSUE THREAD

**Issue by ajayambre** [visjs/vis-timeline#1471]
Our Git Actions build pipeline started failing with the below error.
![image](https://user-images.githubusercontent.com/5074211/172894708-6af9d1f0-f6fd-407b-8ced-b61ef6b053b5.png)

I see 2 recent commits by @renovate-bot that upgraded the xss version from 1.0.11 to 1.0.13
https://github.com/visjs/vis-timeline/commit/62dfb4872c9f0c880d5be82af2d5a0ecd47941b6
https://github.com/visjs/vis-timeline/commit/00f1d82a1f7377c31806ff8b876860c2ac90ffc3

IFilterOptions is imported here - https://github.com/visjs/vis-timeline/blob/6ce2e4075ee437886aa735a70ed13680ad04c17f/types/index.d.ts#L28

We fixed our build by downgrading xss version to 1.0.11
![image](https://user-images.githubusercontent.com/5074211/172898099-51d46313-92a7-4d2e-9297-3064a62f3ec4.png)

**Comment by cloakedninjas** [comment]
Yeah - I reported this https://github.com/visjs/vis-timeline/pull/1020 over a year ago - trying to perform an `npm audit fix` pulls in latest `7.5.1` package and then breaks TypeScript compilation

Got a feeling this repo is starting to die
